source/blog/2018-01-12-designing-for-outcomes-digital-innovation-healthcare.html.markdown

```diff
@@ -0,0 +1,79 @@
+many of the commissioners see it as the best or even only way they could meet the financial, quality and transformational challenges they face. 
+At the same time, a push towards enhanced use of digital technologies is promising to relieve pressure on repetitive administrative 
+tasks and offer better access to patients. Many of the digital solutions that are being implemented, however, are relatively new and lack robust, long-term data necessary to 
+establish definitive evidence for their outcomes. Healthcare providers and commissioners are often reluctant to invest in digital 
+innovation due to a perceived uncertainty, lacking the information necessary to make confident decisions.
+
+Presented below are lessons we’ve learned through our work on digital innovation in healthcare as well as case studies that we recently 
+presented at the *Digital Design: Transforming Healthcare* event at the [Health Foundry](http://www.healthfoundry.org/). 
+
+<br />
+
+# Designing for outcomes – lessons learnt
+
+* **Describe the desired outcomes for patients and stakeholders.** We do this by mapping stakeholders and their outcomes, connecting them into a Theory of Change – 
+  a logical model that describes what we aim to do and why it matters.
+
+* **Look for outcomes, not outputs.** Meaningful impact might not be easy to measure, but measuring outputs is not a good substitute. A service that 
+  processes a higher number of patients is not necessarily better at improving people’s health.
+
+* **Gather evidence appropriate to the stage of innovation.** In the early stages of innovation, this might mean gathering qualitative feedback 
+  from patients and clinicians to understand their underlying needs. Reliable quantitative evidence for innovation is more easily provided in 
+  later stages, by testing a functional prototype with a pilot group. See [Nesta’s Standards of Evidence](https://www.nesta.org.uk/publications/nesta-standards-evidence) for further information about this topic.
+
+* **Keep metrics in mind from the start.** It will be much easier to demonstrate that a service is causing an outcome if measurements are set 
+  up as early on as possible. Measurement points should be designed in a patient-friendly way – as an integrated part of the journey, rather than an unwanted nuisance.
+
+* **Get feedback in a real-world context.** One of the most important elements of designing for outcomes is to keep a connection with the people 
+  and the environment we’re designing for. We do this by creating functional prototypes early and testing them in context with patients and clinical staff. 
+  This provides us with necessary guidance to iterate and improve our digital solutions.
+
+
+![Mapping outcomes](https://s3-eu-west-1.amazonaws.com/unboxed-web-image-uploader/b8c1ec8d373f4c3a4af0b07f884017bc.jpg)
```

Comment:
The bottom part of the image is cropped for me so that the two bottom info box elements are cut off a little - is it possible to avoid this?